charts/osm/templates/osm-configmap.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: osm-config
   namespace: {{ .Release.Namespace }}
```

Comment:
the namespaces in AKS are kube-system, I have left it as is here as I am not sure of how these charts would be deployed

Reply:
We can update all the namespaces to `arc-osm-system`.

Reply:
updated, I have alos changed the name of other resources that that dependency on `Release.Namespace`